scenariorun/src/main/kotlin/com/cosmotech/scenariorun/service/ScenarioRunServiceImpl.kt

```diff
@@ -497,7 +497,9 @@ class ScenarioRunServiceImpl(
     logger.debug(startInfo.toString())
     val scenarioRunRequest =
         workflowService.launchScenarioRun(
-            startInfo.startContainers, startInfo.runTemplate.executionTimeout)
+            startInfo.startContainers,
+            startInfo.runTemplate.executionTimeout,
+            startInfo.solution.alwaysPull!!)
```

Comment:
Is `alwaysPull` nullable with a default value provided in the openapi definition? Do we need `!!`?

Reply:
without !!, Type mismatch: inferred type is Boolean? but Boolean was expected

Reply:
Ok, I admit I did not check, I just though that with a default value there was no need for the property to be nullable.